src/components/my-history/custom-my-history.vue

```diff
@@ -22,6 +22,15 @@
 
             <div class="x-flex x-flex-col x-gap-2">
               <p>{{ suggestion.query }}</p>
+              <p
+                v-if="suggestion.selectedFilters.length > 0"
+                class="x-text1-sm x-text-lead-50 x-line-clamp-1"
+              >
+                <span v-for="filter in suggestion.selectedFilters" :key="filter.id" class="x-pr-8">
+                  {{ filter.label ?? filter.id }}
```

Comment:
```suggestion
                  {{ filter.label }}
```

You can accept this commit by pointing to x-components "3.0.1-alpha.1"